app/controllers/pizzas_controller.rb

```diff
@@ -0,0 +1,12 @@
+class PizzasController < ApplicationController
+  def index
+    @pizzas = Pizza.all
+    @pizza_brands = []
+    
+    @pizzas.each do |pizza|
+      if !@pizza_brands.include?(pizza.brand)
+        @pizza_brands << pizza.brand
+      end
+    end
```

Comment:
Im going allow this an merge it...BUT this would be a fantastic method to define on the Pizza model. Such that you can just call Pizza.brands. I would use the `.pluck` method to help